src/Workleap.Extensions.Mongo.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,15 @@
-#nullable enable
\ No newline at end of file
+#nullable enable
```

Comment:
Sure. I was just auto removing warning with Rider and it moved everything in there. Ill follow your guidance on that.

Reply:
After discussion everything should be in Shipped.txt it will fix the warning, it just the fix is not automatic. Unshipped should only be kept for experimental features which is not the case here.